snippets/base/models.py

```diff
@@ -1099,6 +1110,39 @@ class FxASignupTemplate(Template):
         help_text=('The background color of the button. Valid CSS color. '
                    'Defaults to Firefox Theme Color.'),
     )
+    scene1_section_title_icon = models.ForeignKey(
+        Icon,
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
Let's change this to `models.PROTECT` to prevent users from accidentally removing icons from published snippets. If you can please change this on the simple snippet as well.